homeassistant/helpers/config_validation.py

```diff
@@ -42,6 +42,8 @@ def isfile(value):
 
 def ensure_list(value):
     """Wrap value in list if it is not one."""
+    if isinstance(value, set):
```

Comment:
This changes what we expect ensure_list to do as it no longer wraps sets. Also, a user will never provide a set so I don't really get why this added - please remove.